rails/lib/materials/data_helpers.rb

```diff
@@ -316,7 +316,7 @@ def external_copyable(material)
         return false;
       end
 
-      if material.launch_url.blank?
+      if material.author_url.blank?
```

Comment:
Probably you should check the tool of the resource to see if it has a remote_duplicate_url too.